events/routes.js

```diff
@@ -0,0 +1,10 @@
+const validations = {
```

Comment:
Tempted to have the validation at the same level than the route definition; like https://github.com/ctavan/express-validator#schema-validation

Reply:
Another solution would be to be forwarding down the API validation to the model itself. It becomes an issue for computed fields (date_before & such) (see ? http://vincit.github.io/objection.js/#validation && https://github.com/Vincit/objection.js/blob/master/doc/includes/RECIPES.md#error-handling ?)